src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeedFormatter.cs

```diff
@@ -496,50 +475,40 @@ internal static SyndicationFeed CreateFeedInstance(Type feedType)
 
         internal static void LoadElementExtensions(XmlBuffer buffer, XmlDictionaryWriter writer, SyndicationFeed feed)
         {
-            if (feed == null)
-            {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("feed");
-            }
+            Debug.Assert(feed != null);
+
```

Comment:
I only checked this first overload of LoadElementExtensions and it can receive a null. Please revert the null check exception changes for the LoadElementExtensions methods.